### PR TITLE
fix: nested `<p>` warnings in test and Viewer

### PR DESCRIPTION
**Pull request by tuckerzp**

### CODE REVIEW

File: packages/oscal-react-library/src/components/OSCALControlProse.js [hunk 1]
```diff
@@ -25,9 +25,7 @@ const OSCALStatementEditControlsContainer = styled(Grid)`
   text-align: right;
 `;
 
-const NotImplementedStatement = styled(Typography)(
-  ({ theme }) => `color: ${theme.palette.grey[400]};`
-);
+const NotImplementedStatement = styled(Stack)(({ theme }) => `color: ${theme.palette.grey[400]};`);
```
Comment (kylelaker) [inline packages/oscal-react-library/src/components/OSCALControlProse.js:28]: This should probably not be a `Stack`... In fact, whatever is using this should probably just use the component you created in #790 right?

**Review by Bronstrom — APPROVED** [review]

File: packages/oscal-react-library/src/components/OSCALControlProse.js [hunk 2]
```diff
@@ -309,10 +308,10 @@ export function OSCALReplacedProseWithParameterLabel(props) {
 
   if (!props.isImplemented) {
     return (
-      <NotImplementedStatement>
+      <NotSpecifiedTypography>
```
Comment (kylelaker) [inline packages/oscal-react-library/src/components/OSCALControlProse.js:311]: How does changing from a `Stack` to a `Typography` preserve the correctness of this? Are we sure this doesn't reintroduce the errors? And can you explain to me how/why?

Comment (tuckerzp) [reply packages/oscal-react-library/src/components/OSCALControlProse.js:311]: You're right. I was worried about that. Ideally I'd set the `Typography` component to something like a `<div>`, but that didn't seem to work.

File: packages/oscal-react-library/src/components/OSCALControlProse.js [hunk 3]
```diff
@@ -315,27 +308,27 @@ export function OSCALReplacedProseWithParameterLabel(props) {
...
```
Comment (tuckerzp) [inline packages/oscal-react-library/src/components/OSCALControlProse.js:311]: Something like this does not work. 

```suggestion
      <NotSpecifiedTypography component={div}>
```

**Pull request merged**